cmd/podman/inspect/inspect.go

```diff
@@ -3,11 +3,15 @@ package inspect
 import (
 	"context"
 	"fmt"
+	"html/template"
```

Comment:
I think this should be `text/template`

Reply:
@Luap99 Good catch my IDE auto-imported the wrong package everywhere and I missed that one.